releasetool/commands/start/java.py

```diff
@@ -282,7 +287,7 @@ def determine_last_release(ctx: Context) -> None:
 def gather_changes(ctx: Context) -> None:
     click.secho(f"> Gathering changes since {ctx.last_release_version}", fg="cyan")
     ctx.changes = releasetool.git.summary_log(
```

Comment:
Could you add a click.secho here to let the user know what the source branch is?